web/components/editor.tsx

```diff
@@ -56,6 +63,71 @@ export function useTextEditor(props: {
           class: clsx('no-underline !text-indigo-700', linkClass),
         },
       }),
+      Mention.configure({
+        HTMLAttributes: {
+          class: clsx('not-prose text-indigo-700', linkClass),
+        },
+        // TODO: do a Next link instead of raw <a>
```

Comment:
Is there a reason to prefer the Next Link? I think it does some pre-caching, but also consumes cpu (according to Marshall's profiling).

If it's at all difficult to switch, I just wouldn't.

Reply:
Decided to render the mention using our existing react component (which uses Next Link - though we can change that later if we want).
This was the simplest way to fix certain bugs.